pldoc/static/js/color-contrast.js

```diff
@@ -65,6 +52,20 @@ define([
             });
         },
 
+        rgbaToHex: function(rgba) {
+            var rgbaValue = rgba.match(/^rgba?[\s+]?\([\s+]?(\d+)[\s+]?,[\s+]?(\d+)[\s+]?,[\s+]?(\d+)[\s+]?/i),
```

Comment:
Yikes 😬 Maybe add a comment showing what kind of vale this will match? I'm assuming just `rgba(x, x, x, x)`